python/CMakeLists.txt

```diff
@@ -132,6 +132,16 @@ add_mlir_python_common_capi_library(CUDAQuantumPythonCAPI
 # Instantiation of all Python modules
 ################################################################################
 
+file(GLOB_RECURSE PYTHON_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/cudaq/*.py")
```

Comment:
In addition to the python/cudaq directory, we also have the python/{runtime,tests,utils} directories. Should those be marked as dependencies and copied over as well? Or are they covered by something else already?